scripts/scil_compute_NODDI_priors.py

```diff
@@ -0,0 +1,190 @@
+                   help='Path to the mean diffusivity (MD) volume.')
+
+    g1 = p.add_argument_group('Metrics options')
+    g1.add_argument(
+        '--fa_min', type=float, default='0.7',
+        help='Minimal threshold of FA (voxels above that threshold are '
+             'considered in the single fiber mask). [%(default)s]')
+    g1.add_argument(
+        '--fa_max', type=float, default='0.1',
+        help='Maximal threshold of FA (voxels under that threshold are '
+             'considered in the ventricles). [%(default)s]')
+    g1.add_argument(
+        '--md_min', dest='md_min',  type=float, default='0.003',
+        help='Minimal threshold of MD in mm2/s (voxels above that threshold '
+             'are considered for in the ventricles). [%(default)s]')
+
+    g2 = p.add_argument_group('Regions options')
+    g2.add_argument(
+        '--roi_radius', type=int, default=20,
+        help='Radius of the region used to estimate the priors. The roi will '
+             'be a cube spanning from ROI_CENTER in each direction. '
+             '[%(default)s]')
+    g2.add_argument(
+        '--roi_center', metavar='tuple(3)', nargs="+", type=int,
+        help='Center of the roi of size roi_radius used to estimate the '
+             'priors. [center of the 3D volume]')
+
+    g3 = p.add_argument_group('Outputs')
+    g3.add_argument(
+        '--output_1fiber', metavar='file',
+        help='Output path for the text file containing the single '
+             'fiber average value of AD. If not set, the file will not be '
+             'saved.')
+    g3.add_argument(
+        '--mask_output_1fiber', metavar='file',
+        help='Output path for single fiber mask. If not set, the mask will '
+             'not be saved.')
+    g3.add_argument(
+        '--output_ventricles', metavar='file',
+        help='Output path for the text file containing the ventricles average '
+             'value of MD. If not set, the file will not be saved.')
+    g3.add_argument(
+        '--mask_output_ventricles', metavar='file',
+        help='Output path for the ventricule mask. If not set, the mask will '
+             'not be saved.')
+
+    add_overwrite_arg(p)
+    add_verbose_arg(p)
+
+    return p
+
+
+def load(path):
+    img = nib.load(path)
+    return img.get_fdata(dtype=np.float32), img.affine, img.header.get_zooms()[:3]
+
+
+def save(data, affine, output):
+    img = nib.Nifti1Image(np.array(data, 'float32'),  affine)
+    nib.save(img, output)
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.ad, args.fa, args.md])
+    assert_outputs_exist(parser, args, [],
+                         [args.mask_output_1fiber,
+                         args.mask_output_ventricles,
+                         args.output_ventricles,
+                         args.output_1fiber])
+
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG)
+
+    fa, aff, hdr = load(args.fa)
```

Comment:
I would remove that and do it the slightly longer way (but uniform everywhere in most script)